src/helpers/tests/agGrid.spec.ts

```diff
@@ -309,6 +340,32 @@ describe('getMinimalRowData', () => {
     });
 });
 
+describe('assortDimensionHeaders', () => {
+    it('should return attribute and measure dimension headers', () => {
+        const dimensions = fixtures.pivotTableWithColumnAndRowAttributes.executionResponse.dimensions;
+        const { attributeHeaders, measureHeaderItems } = assortDimensionHeaders(dimensions);
+        expect(attributeHeaders).toHaveLength(4);
+        expect(attributeHeaders.filter(header => Execution.isAttributeHeader(header))).toHaveLength(4);
+        expect(measureHeaderItems).toHaveLength(4);
+        expect(measureHeaderItems.filter(header => header.hasOwnProperty('measureHeaderItem'))).toHaveLength(4);
+    });
+});
+
+describe('assignSorting', () => {
+    const ASC = 'asc';
```

Comment:
use constant from import

Reply:
this is also by design. We should bring this up on FE sync.